workflow/utils.py

```diff
@@ -0,0 +1,116 @@
+
+import datetime
+import os
+from pathlib import Path
+
+import pandas as pd
+
+# BIDS format
+BIDS_SUBJECT_PREFIX = 'sub-'
+BIDS_SESSION_PREFIX = 'ses-'
+
+# directory/file names
+DNAME_BACKUPS_MANIFEST = '.manifests'
+DNAME_BACKUPS_STATUS = '.doughnuts'
+FNAME_MANIFEST = 'mr_proc_manifest.csv'
+FNAME_STATUS = 'doughnut.csv'
+
+# for creating backups
+TIMESTAMP_FORMAT = '%Y%m%d_%H%M'
+EXT_SYMBOL = '.'
+SEP_FNAME_BACKUP = '-'
+
+# manifest file columns
+COL_SUBJECT_MANIFEST = 'participant_id'
+COL_BIDS_ID_MANIFEST = 'bids_id'
+COL_VISIT_MANIFEST = 'visit'
+COL_SESSION_MANIFEST = 'session'
+COL_DATATYPE_MANIFEST = 'datatype'
+COLS_MANIFEST = [COL_SUBJECT_MANIFEST, COL_BIDS_ID_MANIFEST, COL_VISIT_MANIFEST, 
+                 COL_SESSION_MANIFEST, COL_DATATYPE_MANIFEST]
+
+# status file columns
+COL_DICOM_DIR = 'participant_dicom_dir'
+COL_DICOM_ID = 'dicom_id'
+COL_DOWNLOAD_STATUS = 'downloaded'
+COL_ORG_STATUS = 'organized'
+COL_CONV_STATUS = 'converted'
```

Comment:
why not make these `True` or `False`?

Reply:
These are just the column names, the actual values are `True` or `False`